### DISCUSSION TITLE
scanning not as good as dedicated qr app.

### DISCUSSION THREAD

**Discussion by DevDianDankie** [capacitor-community/barcode-scanner#299] [Q&A]
I am noticing that the scanner does not pick up qr codes as well as other QR scanning apps.

I am currently playing around with Lazer engraving QR codes and I just can not get my app to pick up the qr codes on the wood.

But when I use a different app or just my camera the qr code is picked up nicely.

I am wondering why? and what can be done to improve the detection success rate of the plugin?

**Comment by thegnuu** [comment]
@DevDianDankie You should check out https://github.com/capawesome-team/capacitor-mlkit/tree/main/packages/barcode-scanning, it should provide way better results in your case!

**Reply by DevDianDankie** [reply]
Thank you! will have a look.
